variables.tf

```diff
@@ -274,9 +274,14 @@ variable "ssh_user" {
   default     = "root"
 }
 
+variable "ssh_password" {
+  description = "Password which terraform will use to connect to newly created VMs during provisioning"
+  default     = ""
+}
+
 variable "ssh_keyfile" {
   description = "Location of private ssh key to connect to newly created VMs during provisioning"
-  default     = "~/.ssh/id_rsa"
+  default     = "/dev/null"
```

Comment:
using `/dev/null` might cause problems for windows users running the script from their laptop.  although we don't believe that's really a good practice to run terraform from your laptop, for tire-kickers this is likely going to be where they're going to be provisioning sandbox environments.

Reply:
Will play with it